plugins/discourse-lazy-videos/lib/lazy-videos/lazy_youtube.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "onebox"
+
+class Onebox::Engine::YoutubeOnebox
+  include Onebox::Engine
+  alias_method :default_onebox_to_html, :to_html
+
+  def to_html
+    if SiteSetting.lazy_videos_enabled && SiteSetting.lazy_youtube_enabled && video_id &&
+         !params["list"]
+      result = parse_embed_response
+      result ||= get_opengraph.data
+
+      thumbnail_url = "https://img.youtube.com/vi/#{video_id}/maxresdefault.jpg"
+      thumbnail_response = Net::HTTP.get_response(URI(thumbnail_url))
```

Comment:
Shouldn't we use `Onebox::Helpers.fetch_response` instead?

Reply:
Yeah sure